packages/playground-server/src/db.ts

```diff
@@ -328,6 +329,31 @@ export async function bindMultisigToUser(
   }
 }
 
+export async function bindTransactionHashToUser(
+  user: User,
+  transactionHash: string
+): Promise<boolean> {
+  const db = getDatabase();
+
+  const query = db("users")
+    .where({ id: user.id })
+    .update("transaction_hash", transactionHash);
+
+  try {
+    await query;
+
+    Log.debug("Executed createUser query", {
```

Comment:
Minor correction: Not creating a user here, just updating `transaction_hash` property

Reply:
Oh, snap :P I'll fix that.